modules/pagerduty-service-integration-email/generic-email-inbound-integration.tf

```diff
@@ -0,0 +1,21 @@
+resource "pagerduty_service_integration" "generic_email_inbound_integration" {
+  name              = var.name
+  service           = var.service_id
+  integration_email = var.integration_email
+  # hardcoded now since we don't support multiple email filters.
+  email_filter_mode = "and-rules-email"
+  # hardcoded because this is a generic_email_inbound_integration module.
+  type                    = "generic_email_inbound_integration"
+  email_incident_creation = var.email_incident_creation
+
+  email_filter {
+    # v1 we don't expose this yet.
```

Comment:
comment is not so clear. Better to say:

"In the initial implementation of this module, we hardcode this value..."
"Future enhancement: expose this attribute as a variable..."